src/components/TagMenu/index.tsx

```diff
@@ -102,18 +101,13 @@ export function TagMenu({
                 ]}>
                 <Link
                   label={_(msg`Search for all posts with tag ${displayTag}`)}
```

Comment:
Maybe we should change the label to "View all posts with tag ${displaytag}" since it doesn't push to search anymore?

Reply:
Good call, updated!